tests/functional/prometheous/test_prometheus_validations_file.py

```diff
@@ -333,3 +335,67 @@ def test_prometheus_pv_resize(self):
             pod_name=pod_name, pvc_name=pvc_name,
             filename="secondfilename", dirname="seconddirname",
             metric_data=resize_metrics, operation="create")
+
+    @pytest.mark.tier4
+    def test_prometheus_volume_metrics_on_node_reboot(self):
+        """Validate volume metrics using prometheus before and after node
+        reboot"""
+
+        # Pod name for the entire test
+        prefix = "autotest-{}".format(utils.get_random_str())
+
+        # Create I/O pod with PVC
+        pvc_name = self.create_and_wait_for_pvc()
+        pod_name = openshift_ops.oc_create_tiny_pod_with_volume(
+            self._master, pvc_name, prefix,
+            image=self.io_container_image_cirros)
+        self.addCleanup(openshift_ops.oc_delete, self._master, 'pod', pod_name,
+                        raise_on_absence=False)
+        openshift_ops.wait_for_pod_be_ready(
+            self._master, pod_name, timeout=60, wait_step=5)
+
+        # Write data on the volume and wait for 2 mins and sleep is must for
+        # prometheus to get the exact values of the metrics
+        ret, _, err = openshift_ops.oc_rsh(
+            self._master, pod_name, "touch /mnt/file{1..1000}")
+        time.sleep(120)
+
+        # Fetch the metrics and store in initial_metrics as dictionary
+        initial_metrics = self._get_and_manipulate_metric_data(
+            self.metrics, pvc_name)
+        openshift_ops.switch_oc_project(
```

Comment:
Add a cleanup for the project switch

Reply:
ack will do.

Reply:
Done

Reply:
This is incorrect cleanup. Step is to switch to `storage_project_name` and cleanup is also for `storage_project_name` which makes this cleanup absolete.

In this case need to get current peoject name and same as cleanup. If current project name is `storage_project_name` then no need to add cleanup.

Reply:
Done